### PR TITLE
Recursive GCD test

### PR DESCRIPTION
**Pull request by rxie25**
added and skipped recursive gcd test.

### CODE REVIEW

**Comment by rxie25** [comment]
@jzacsh take a look at the test case.

File: src/tests/test-recursive-gcd.skip.eq [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+   else {
+     a = b % a;
+     b = a;
+   }
+   gcd;
+   }
+}
+
+Gcd:find gcd {
+  print("gcd between 10 and 20 is %0.f\n", gcd);
```
Comment (jzacsh) [inline src/tests/test-recursive-gcd.skip.eq]: please declare `a = 10` and `b = 20` down here in your `find` decl, for clarity

File: src/tests/test-recursive-gcd.skip.eq [hunk 2]
```diff
@@ -0,0 +1,29 @@
+Gcd = {
+  a = 10;
+  b = 20;
+  gcd = {
```
Comment (jzacsh) [inline src/tests/test-recursive-gcd.skip.eq]: @nnhoang want to take a look? I know you sometimes don't like how the GCD examples are written up

**Comment by jzacsh** [comment]
looks good.. comments inside

**Pull request merged**